_episodes/convert.md

```diff
@@ -16,3 +16,39 @@ learners should be able to
 __convert a shell script into a CWL workflow__
 
 {% include links.md %}
+
+```bash=
+#!/usr/bin/env bash
+
+if ! [ -f data/reference.fa.sa ]; then
+   bwa index data/reference.fa
+fi
+
+for sample in samples; do
+   bwa mem data/reference.fa $sample > outputs/$sample.sam
+done
+
+# we have limited storage space so we
+# cleanup indices once we are done
+rm -f data/reference.fa.*
+
+# TODO: more commands, to demonstrate data linkages
+
+```
+
+Exercise: Find existing CWL tool descripts for all the tools in the above script
```

Comment:
"CWL tool descripts" should probably be "CWL tool descriptions"

Reply:
Probably should explain why they should look in the following github directory and explain what it is , unless you are explaining that above in the lesson.

Reply:
yes, the lesson text (to be written later) should give the necessary information. We can outline the needed pieces now